Task_1/Task_1/src/components/to_do_list/task_bar.jsx

```diff
@@ -0,0 +1,18 @@
+
+import moment from 'moment'; 
+const TaskBar = (props) => {
+
+    let date=props.date;
+    let day = moment(date);
+
+  return (
+    <div className="mt-[20px] w-[900px] bg-indigo-300 mx-auto rounded-2xl \ text-2xl border-2 border-indigo-700 font-bold  mb-2">
```

Comment:
mt-[20px] there are predefined classes, we can use those. if need custom classes add config variable and use it.

Reply:
Done